Adapter_oM/IBHoMAdapter.cs

```diff
@@ -37,6 +38,7 @@ public interface IBHoMAdapter
         ModuleSet AdapterModules { get; }
         Dictionary<Type, object> AdapterComparers { get; }
         Dictionary<Type, List<Type>> DependencyTypes { get; }
+        Dictionary<Type,int> PriorityTypes {get;}
```

Comment:
Think this should simply be a List of types that should come before all others, and for them to simply be pushed in the order defined in that list.

```suggestion
        List<Type> PriorityTypes {get;}
```

Reply:
Agree with @IsakNaslundBh on this one.

Reply:
@IsakNaslundBh , thanks for the comment.
Updated the code in line with it.